store/tx_manager.go

```diff
@@ -330,19 +335,41 @@ func (txm *EthTxManager) ContractLINKBalance(wr models.WithdrawalRequest) (asset
 	return *linkBalance, nil
 }
 
+// transact converts arguments for a method in a contract to a byte array that can be used to interact with the contract
+func transact(contractName string, method string, args ...interface{}) ([]byte, error) {
+	jsonFile, err := os.Open("../solidity/build/contracts/" + contractName + ".json")
```

Comment:
These build files only exist there relative to chainlink executable in the source code, they won't exist there once you build the executable. Let's sync up over a call to talk through possible solutions.